app/models/org_space_policy.rb

```diff
@@ -0,0 +1,75 @@
+require 'conjur_client'
+
+class OrgSpacePolicy
+  class OrgPolicyNotFound < RuntimeError
+  end
+
+  class SpacePolicyNotFound < RuntimeError
+  end
+
+  class SpaceLayerNotFound < RuntimeError
+  end
+
+  class << self
+    def ensure_exists(org_id, space_id)
+      OrgSpacePolicy.new(org_id, space_id).ensure_exists
```

Comment:
is it correct that until your provision changes get in, the bind request will fail because they won't have org / space policies set up?

Reply:
It is true it will fail unless the policy is loaded before bind. This is documented in the README now as a manual step, and will be handled automatically by the service broker after the provision changes.